apps/backend/src/allowlist/pipes/sign-message.pipe.ts

```diff
@@ -0,0 +1,56 @@
+import {
+  PipeTransform,
+  Injectable,
+  ArgumentMetadata,
+  BadRequestException,
+} from '@nestjs/common';
+import { StdSignature, verifyNonceMsgSigner } from 'cudosjs';
+import { SignedMessageDto } from '../dto/signed-message.dto';
+
+@Injectable()
+export class SignMessagePipe implements PipeTransform {
+  async transform(value: SignedMessageDto, metadata: ArgumentMetadata) {
+    if (
+      !value.signature ||
+      !value.address ||
+      !value.message ||
+      !value.sequence ||
+      !value.account_number ||
+      !value.chain_id
+    ) {
+      throw new BadRequestException('Missing data');
+    }
+
+    const val = {};
+    Object.keys(value).forEach((key) => {
+      if (key === 'sqeuence' || key === 'account_number') {
+        val[key] = +value[key];
+      } else if (key === 'signature') {
```

Comment:
`else if` structure is a bad practice since is really hard to follow, please try to remvoe it and use only if's or other patterns